helix-term/src/commands.rs

```diff
@@ -2783,6 +2786,7 @@ fn symbol_picker(cx: &mut Context) {
                         ));
                         Some((path, line))
                     },
+                    |_, l_score, _, r_score| l_score.cmp(&r_score).reverse(),
```

Comment:
Why do we need to do this everywhere? Why not just have it sorted in the order it will be displayed?

Reply:
This sort function specifies the order it is displayed in. If we'd replace this with a different sort function, the matches would no longer be sorted by their matching score, so we need this line to mimick the old behavior.
I thought about making the sort function parameter an Option and using this as a default if None is given. I don't think that's a great idea though, especially because most pickers can probably be improved by adding additional criteria to their sort functions.